### PR TITLE
Be able to work on Win10

### PR DESCRIPTION
**Pull request by hyde777**
Don't forget to stop the Hyper-V and target a working ssh bash comman…d(like the git one)

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+# config.ssh.private_key_path = "C:/Users/'Yohan FAIRFORT'/.vagrant.d/insecure_private_key"
```
Comment (alex-lairan) [inline Vagrantfile:3]: ?

File: Vagrantfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
```
Comment (alex-lairan) [inline Vagrantfile:4]: Trailing White Line

File: Vagrantfile [hunk 3]
```diff
@@ -9,6 +11,10 @@ Vagrant.configure(2) do |config|
 
   config.vbguest.auto_update = false if defined?(VagrantVbguest::Middleware)
   config.ssh.forward_agent = true
+
+  if ENV['OS'] == "Windows_NT"
```
Comment (alex-lairan) [inline Vagrantfile:15]: `"Windows_NT"` should be `'Windows_NT'`

**Review by alex-lairan — CHANGES_REQUESTED** [review]
Déso je suis chiant mais quand on touche à du ruby, c'est sacré pour moi <3

**Comment by hyde777** [comment]
Even after this there is some other problem.
We should just abort this branch

**Pull request closed without merging**